src/confluent_kafka/src/Consumer.c

```diff
@@ -1409,6 +1436,20 @@ static PyMethodDef Consumer_methods[] = {
           "  :raises: RuntimeError if called on a closed consumer\n"
           "\n"
         },
+        { "memberid", (PyCFunction)Consumer_memberid,
+          METH_VARARGS|METH_KEYWORDS,
```

Comment:
As it doesn't take any arguments it should have METH_NOARGS or whatever it is called.